docs/source/reservations/reservations.rst

```diff
@@ -41,21 +45,26 @@ You can specify two fields in ``~/.sky/config.yaml``:
       specific_reservations:
         # 1x H100 capacity block in us-east-1
         - "cr-0123456789"
-        # 2x A100 reservation in us-west-2
+        # 2x A100 reservation in us-east-2
         - "cr-123456789a"
-        # 2x M5a.16xlarge reservation in us-east-1
+        # 2x A100 reservation in us-west-2
         - "cr-23456789ab"
+        # 2x M5a.16xlarge reservation in us-east-1
+        - "cr-3456789abc"
 
-For more details of the configuration, see reference for :ref:`config-yaml`.
+For more details of the fields, see :ref:`config-yaml`.
 
+.. note::
+
+    If any of the fields are specified, please allow about 30 seconds for SkyPilot optimizer to retrieve the latest reservation/block status on all regions and zones from your AWS account.
```

Comment:
style: No need to use "Please" in tech docs / print messages.

"....SkyPilot optimizer may take around 30 seconds to ..."